dxr/plugins/python/indexers.py

```diff
@@ -260,10 +260,11 @@ def analyze_tokens(self):
                 # names, we start the queue for the current parenthesis level
                 # at the byte offset for the keyword token, but only start
                 # pushing character offsets once we're past the keyword.
-                paren_stack.setdefault(paren_level, (utf8_start, []))
+                queue_start, node_queue = paren_stack.setdefault(paren_level, (utf8_start, []))
+                node_start_table.setdefault(queue_start, node_queue)
```

Comment:
Do I deduce correctly that, if there's already a `node_start_table[queue_start]` key, we can count on it pointing to the same `node_queue` value, so it doesn't matter if this does nothing?